development/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG NAUTOBOT_VER="1.4.1"
+ARG NAUTOBOT_VER="1.4.7"
```

Comment:
Is this some requirement? or just to update it?

Reply:
Just to update to a version that works for the bug found in Nautobot